cmd/metrics-adapter/app/options/options.go

```diff
@@ -64,7 +72,10 @@ func (o *Options) Complete() error {
 func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	o.CustomMetricsAdapterServerOptions.AddFlags(fs)
 	o.ProfileOpts.AddFlags(fs)
-
+	fs.Float32Var(&o.ClusterAPIQPS, "cluster-api-qps", 40.0, "QPS to use while talking with cluster kube-apiserver. Doesn't cover events and node heartbeat apis which rate limiting is controlled by a different set of flags.")
+	fs.IntVar(&o.ClusterAPIBurst, "cluster-api-burst", 60, "Burst to use while talking with cluster kube-apiserver. Doesn't cover events and node heartbeat apis which rate limiting is controlled by a different set of flags.")
+	fs.Float32Var(&o.KubeAPIQPS, "kube-api-qps", 40.0, "QPS to use while talking with karmada-apiserver. Doesn't cover events and node heartbeat apis which rate limiting is controlled by a different set of flags.")
+	fs.IntVar(&o.KubeAPIBurst, "kube-api-burst", 60, "Burst to use while talking with karmada-apiserver. Doesn't cover events and node heartbeat apis which rate limiting is controlled by a different set of flags.")
```

Comment:
I bet it is copied somewhere from Karmada ;)

> Doesn't cover events and node heartbeat apis which rate limiting is controlled by a different set of flags.

And I guess this part is from `kubelet`(see [here](https://github.com/kubernetes/kubernetes/blob/06a81d13959e0e960eb726e98bc1f444d7c36f12/cmd/kubelet/app/options/options.go#L489)), and that is not applicable for Karmada as we don't do node heartbeat. :)

Reply:
So, do you mean these should be deleted?

Reply:
> So, do you mean these should be deleted?

Yes, probably all of them from the Karmada repo.

Reply:
Done.